target/snitch_cluster/README.md

```diff
@@ -244,8 +244,8 @@ The traces contain a lot of information which we might not be interested at firs
 
 ```
             , compute
-"range(0,9)",       1
-9           ,
+"range(0,8)",       1
+8           ,
 
 ```
 
```

Comment:
The following two lines need to remove one hierarchy `../`:

```
# Labels, filters and reorders the event regions as specified by an application-specific layout file
../../util/trace/layout_events.py logs/event.csv sw/apps/axpy/layout.csv -o logs/trace.csv
# Creates a trace file which can be visualized with Chrome's TraceViewer
../../util/trace/eventvis.py -o logs/trace.json logs/trace.csv
```

Reply:
It is 2-3 lines below the `layout.csv` example in the same documentation file `target/snitch_cluster/README.md`. As it is not in the "modified code" area, I can not mark it exactly, but when testing it I noticed the instructions only need two `../../util....` instead of three `../../../util.....`. Thought it might make sense to fix it in this PR aswell :)

Reply:
Good catch! Thanks :)

I made the suggested corrections.